### DISCUSSION TITLE
DNSCrypt-Proxy 2.1.1 [Anonymized DNS] - Stops resolving after a few days

### DISCUSSION THREAD

**Discussion by Jwzzlf** [DNSCrypt/dnscrypt-proxy#1996] [Potential issues]
Can anyone inform me how I can fix this problem? I have about 20 Anonymized DNS public servers setup via Anonymized DNS relays which worked perfectly for about a couple of days.. [./dnscrypt-proxy -check gives me a correct config]

Unfortunatly DNSCrypt-proxy stopped resolving.. Normally it starts working again after restarting the service but it still can resolve.

Can anyone point me in the right direction how to fix this issue? 

Any helpfull information would be much appreciated

**Comment by BrianG61UK** [comment]
What hardware is it running on? Are you using DoH, DNSCrypt or what?
